### ISSUE TITLE
SpriteList.shuffle() is not shuffling the order of the list

### ISSUE THREAD

**Issue by yegarti** [pythonarcade/arcade#1010]
## Bug Report
`SpriteList.shuflle()` does not seem to shuffle the order of the list.

### Actual behavior:
Sprites stay in the same order

### Expected behavior:
Sprites shuffled.

### Steps to reproduce/example code:
```python
import arcade

sprite_list = arcade.SpriteList()
for i in range(20):
    sprite_list.append(arcade.SpriteSolidColor(1, 1, arcade.color.GREEN))
old_ids = [id(s) for s in sprite_list]
sprite_list.shuffle()
new_ids = [id(s) for s in sprite_list]

print(old_ids != new_ids)  # False
```

### Environment: 
```
Python 3.9.7
arcade 2.6.3
```
Thanks.

**Comment by einarf** [comment]
Thanks a lot for reporting. Fixed in bad8ed764ee43c9fc7f3c0a0f163b330e511beea

**Comment by einarf** [comment]
2.6.4 released
